addons/binding/org.openhab.binding.zwave/ESH-INF/thing/fibaro_fgbs001_0_0.xml

```diff
@@ -73,6 +73,8 @@
       <property name="model">FGBS001</property>
       <property name="manufacturerId">010F</property>
       <property name="manufacturerRef">0501:0102,0501:1002,0501:3002</property>
+      <property name="commandClass:SENSOR_ALARM:1"></property>
```

Comment:
Hi,
Sorry for the delays - I’ve been on the road all week (and in France - so travel has not been fun or easy!!!). I’ll try and catch up with things today/tomorrow…

Cheers
Chris


> On 25 May 2016, at 17:28, jongj <notifications@github.com> wrote:
> 
> In addons/binding/org.openhab.binding.zwave/ESH-INF/thing/fibaro_fgbs001_0_0.xml <https://github.com/openhab/openhab2-addons/pull/966#discussion_r64603340>:
> 
> > @@ -73,6 +73,8 @@
> >        <property name="model">FGBS001</property>
> >        <property name="manufacturerId">010F</property>
> >        <property name="manufacturerRef">0501:0102,0501:1002,0501:3002</property>
> > +      <property name="commandClass:SENSOR_ALARM:1"></property>
> both entries are missing supportedGetSupported=false. I send you a PM with a request if you could add this to the database but I am not sure if you received it ....
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub <https://github.com/openhab/openhab2-addons/pull/966/files/1c8f6c895b11c58c73dfd674cf02181907938abb#r64603340>